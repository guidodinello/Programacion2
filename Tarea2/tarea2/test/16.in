# Implementar pertenece
pertenece 5
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (6,2.6)
pertenece 5
pertenece 6
pertenece 8
pertenece 9
Fin
