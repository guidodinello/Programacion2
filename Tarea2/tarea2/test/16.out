1>#  Implementar pertenece.
2>5 no pertenece a cad.
3>Segmento insertado despuÃ©s de loc.
4>5 pertenece a cad.
5>6 pertenece a cad.
6>8 pertenece a cad.
7>9 no pertenece a cad.
8>Fin.
